public/stylesheets/site/2.0/18-zone-searchbrowse.css

```diff
@@ -103,6 +107,12 @@ form.filters dl {
   padding-right: 0;
 }
 
+.filters .submit input {
```

Comment:
Stops the "Sort and Filter" text from being cut off when the filters are narrow, as seen in the above screen shot. Now the text can wrap across multiple lines if necessary.